### PR TITLE
Adding CLI documentation for semi-technical users, distribution via cloning the repo and symlinking the `cli_entry.sh` to global directory

### PR DESCRIPTION
**Pull request by nicholaspai**
Assumption is that users know how to install via the command line a package. After that, installation and starting of the CLI tool should be seamless.

Signed-off-by: Nick Pai <npai.nyc@gmail.com>

### CODE REVIEW

**Comment by mrice32** [comment]
@nicholaspai can we move this to documentation/tutorials or maybe symlink it to put it in both places?

**Comment by nicholaspai** [comment]
> @nicholaspai can we move this to documentation/tutorials or maybe symlink it to put it in both places?

Great idea, done

File: documentation/tutorials/using_the_voting_cli.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Voting Command Line Interface (CLI)
+
+We provide a user-friendly command line interface for voting on price requests and retrieving rewards.
+
+# Setup
```
Comment (mrice32) [inline documentation/tutorials/using_the_voting_cli.md]: One super minor nit: a heading followed immediately by a subheading doesn't look great (and is a bit confusing) when rendered on the docs site: https://28293-142203027-gh.circle-artifacts.com/0/build/site/uma/tutorials/using_the_voting_cli.html#_setup. I would recommend either adding a little text between to introduce the `Setup` section or promoting the subheadings and removing `Setup` altogether.

Comment (nicholaspai) [reply documentation/tutorials/using_the_voting_cli.md]: done

**Pull request merged**